site/src/docs/components/header/index.mdx

```diff
@@ -75,3 +75,8 @@ Coming soon!
 - Using the Header.SkipLink component is optional. Use the skip link component to help keyboard-only users skip to the main content on a page.
 - See example for Header.SkipLink <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-skip-link">in Storybook</Link>
 - The Header.SkipLink supports ariaLabel which gives possibility to define more informative text for screen readers
+
+#### With theme
+
+- Header supports two themes: light (default) and dark. See example for light theme <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features">in Storybook</Link> and for dark theme <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features-dark-theme">in Storybook</Link>
```

Comment:
You could just add the example here inside a <PlaygroundPreview /> so it's clear what it looks like straightaway.